modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc

```diff
@@ -1,12 +1,10 @@
 // Module included in the following assemblies:
 //
-// * security/cert_manager_operator/cert-manager-authenticate-gcp.adoc
+// * security/cert_manager_operator/cert-manager-authenticate.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="cert-manager-configure-cloud-credentials-gcp-sts_{context}"]
-= Configuring cloud credentials for the {cert-manager-operator} with GCP Workload Identity
-
-Generate the cloud credentials for the {cert-manager-operator} by using the `ccoctl` binary. Then, apply them to the GCP Workload Identity cluster.
+= Authenticating with GCP Workload Identity
```

Comment:
Same comment here, but I was also wondering your thoughts

Reply:
I scanned through multiple topics in our docs. Almost all of them directly refer to these as AWS or GCP,hardly the expanded forms. I would be inclined to stick to this format to keep it consistent.